debian/patches/cpick-eee60329-Fix-cloud-init-status-wait-when-no-datasource-found

```diff
@@ -0,0 +1,169 @@
+From eee603294f120cf98696351433e7e6dbc9a3dbc2 Mon Sep 17 00:00:00 2001
+From: James Falcon <james.falcon@canonical.com>
+Date: Wed, 23 Mar 2022 18:03:00 -0500
+Subject: [PATCH] Fix cloud-init status --wait when no datasource found (#1349)
+
+* Fix cloud-init status --wait when no datasource found
+
+In 0de7acb1, we modified status checks to wait until we get an "enabled"
+or "disabled" file from ds-identiy. ds-identify never outputs a
+"disabled" file, so "status --wait" will wait indefinitely if no
+datasource is found.
+
+LP: #1966085
```

Comment:
It is great that you already have `LP: #1966085` but for formal style (https://dep-team.pages.debian.net/deps/dep3/) you'd usually refer those like:

```
Origin: https://github.com/canonical/cloud-init/commit/eee60329
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1966085
Last-Update: 2022-03-24
```

In fact since you needed to modify it here it would be

```
Origin: backport, https://github.com/canonical/cloud-init/commit/eee60329
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1966085
Last-Update: 2022-03-24
```


If ever in doubt that link above has all you'd ever want to express in a patch :-)